ipaplatform/redhat/services.py

```diff
@@ -63,8 +63,12 @@
 # Our PKI instance is pki-tomcatd@pki-tomcat.service
 redhat_system_units['pki-tomcatd'] = 'pki-tomcatd@pki-tomcat.service'
 redhat_system_units['pki_tomcatd'] = redhat_system_units['pki-tomcatd']
+redhat_system_units['ipa-custodia'] = 'ipa-custodia'
```

Comment:
I think you need to have the ``.service`` extension here, so ``ipa-custodia.service``.

Reply:
The latest version does not touch redhat/services.py